testing/integration_test.go

```diff
@@ -34,11 +34,12 @@ import (
 )
 
 var (
-	repo   = flag.String("repo", "https://github.com/GoogleCloudPlatform/cloud-profiler-nodejs.git", "git repo to test")
-	branch = flag.String("branch", "", "git branch to test")
-	commit = flag.String("commit", "", "git commit to test")
-	pr     = flag.Int("pr", 0, "git pull request to test")
-	runID  = strings.Replace(time.Now().Format("2006-01-02-15-04-05.000000-0700"), ".", "-", -1)
+	repo        = flag.String("repo", "https://github.com/GoogleCloudPlatform/cloud-profiler-nodejs.git", "git repo to test")
```

Comment:
GoogleCloudPlatform -> googleapis while we are here?

Reply:
Done.